specification/columns/commitmentdiscountquantity.md

```diff
@@ -10,17 +10,17 @@ The CommitmentDiscountQuantity column adheres to the following requirements:
 
 * CommitmentDiscountQuantity MUST be present in a FOCUS dataset when the provider supports *commitment discounts*.
 * CommitmentDiscountQuantity MUST be of type Decimal and MUST conform to [Numeric Format](#numericformat) requirements.
-* CommitmentDiscountQuantity MAY be negative if [*ChargeClass*](#chargeclass) is "Correction".
+* CommitmentDiscountQuantity MAY be null or any valid decimal value if [*CommitmentDiscountId*](#commitmentdiscountid) is not null and [*ChargeClass*](#chargeclass) is "Correction".
```

Comment:
```suggestion
* CommitmentDiscountQuantity MAY be null or a positive decimal in some [*ChargeClass*](#chargeclass) "Correction" scenarios where [*CommitmentDiscountId*](#commitmentdiscountid) is not null.
```
This is a nitpick. This as is currently written doesn't make sense to me as a requirement. I don't have a great suggestion for how to fix it. It seems unnecessary to have a requirement that says the value can be any allowed value (which for a nullable decimal column is any decimal OR null). Should we only call out abnormal cases?

Fine if you want to ignore this as well.

Reply:
Discussed in TF-1 today, going to revisit a more wide-ranging readability topic in a later release